### PR TITLE
Wrong credentials test on login page

### PR DESCRIPTION
**Pull request by mohdhaider07**
I have added new test on login page "wrong credentials"

### CODE REVIEW

File: functions/auth.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const admin = require("firebase-admin");
 const serviceAccount = require("./private/cl-dev-pk.json");
 admin.initializeApp({
   credential: admin.credential.cert(serviceAccount),
-  databaseURL: process.env.VITE_APP_DATABASE_URL,
+  databaseURL: import.meta.env.VITE_APP_DATABASE_URL
```
Comment (Sarfraz-droid) [inline functions/auth.js:6]: @mohdhaider07 is this working? I don't think that it should. Since the Firebase Functions run on a serverless instance and not in vite.

Just show that the tests are working if possible

File: src/components/Card/CardWithPicture1.jsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
+import React, { useState } from "react";
```
Comment (Sarfraz-droid) [inline src/components/Card/CardWithPicture1.jsx:1]: @mohdhaider07 `CardWithPicture1.jsx` and `CardWithoutPicture1.jsx` aren't being used in Credentials Tests. 

Please remove this from this files as it uses the mui v4.

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

File: functions/auth.js (hunk 1 above)
Comment (mohdhaider07) [reply functions/auth.js:6]: It is working on my pc. Should I undo ?

Comment (mohdhaider07) [reply functions/auth.js]: image

File: cypress/fixtures/login.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-  "email": "sougatariju13@gmail.com",
-  "password": "123456"
+  "email": "haiderahmed12786@gmail.com",
+  "password": "Haider@123"
```
Comment (Sarfraz-droid) [inline cypress/fixtures/login.json]: Revert the changes in fixtures

**Review by Sarfraz-droid — APPROVED** [review]

File: cypress/fixtures/login.json (hunk 3 above)
Comment (mohdhaider07) [reply cypress/fixtures/login.json]: Reverted
